### PR TITLE
feat: updated the design for Messaging Queue - summary section

### PR DESCRIPTION
**Pull request by SagarRajput-7**
#### Screenshots

Before - 
---------
image

Updated design -
------------------ 
image

Interaction and redirection - 
-----------------------
https://github.com/user-attachments/assets/2af601ac-b8e1-4ba4-b666-5e6c39e404fd

#### Affected Areas and Manually Tested Areas

- Tested the redirection to details page
- Compared the prev and new interaction for usability

----

> [!IMPORTANT]
> Updated Messaging Queue summary section with new Kafka metric views and enhanced UI components.
> 
>   - **Design and Functionality**:
>     - Updated `messagingQueuesKafkaOverview.json` with new titles and descriptions for consumer lag, producer latency, partition latency, and drop rate views.
>     - Added `mqServiceView` to `QueryParams` in `query.ts` for handling view-specific queries.
>     - Modified `MQDetailPage.tsx` to use `mqServiceView` for view selection and added drop rate options.
>   - **UI Components**:
>     - Updated `MessagingQueues.tsx` to include new overview cards for consumer, producer, partition, and drop rate views with redirection.
>     - Adjusted styles in `MessagingQueues.styles.scss` for new layout and card designs.
>   - **Utilities**:
>     - Updated `MessagingQueuesUtils.ts` to include new view types in `MessagingQueuesViewType`.
> 
> <sup>This description was created by </sup>[Ellipsis](https://www.ellipsis.dev?ref=SigNoz%2Fsignoz&utm_source=github&utm_medium=referral)<sup> for 6a8c8ae341b4b9849f967f4a7fadf6c540fea716. It will automatically update as commits are pushed.</sup>

### CODE REVIEW

File: frontend/src/pages/MessagingQueues/MQDetailPage/MQDetailPage.tsx
Comment (ahmadshaheer) [inline frontend/src/pages/MessagingQueues/MQDetailPage/MQDetailPage.tsx:1]: can you remove the unused `setSelectedView`?

**Review by ahmadshaheer — DISMISSED** [review]

File: frontend/src/pages/MessagingQueues/MQDetailPage/MQDetailPage.tsx
Comment (SagarRajput-7) [reply frontend/src/pages/MessagingQueues/MQDetailPage/MQDetailPage.tsx:1]: added logic, got missed in resolving conflicts

**Comment by ahmadshaheer** [comment]
@SagarRajput-7, can you check this? there seems to be some issue while switching between views. even if i select **partition latency view**, it keeps **consumer lag view** as active

https://github.com/user-attachments/assets/f17aa848-b271-4972-90ed-5aff60043fe6

**Comment by SagarRajput-7** [comment]
> @SagarRajput-7, can you check this? there seems to be some issue while switching between views. even if i select **partition latency view**, it keeps **consumer lag view** as active
> 
>  consumer.lag.view.-.switching.views.issue.mov

Fixed - 

https://github.com/user-attachments/assets/db68e22f-6101-4297-bb6c-2109fd7cde72

**Review by rahulkeswani101 — DISMISSED** [review]

**Review by ahmadshaheer — APPROVED** [review]

**Review by YounixM — APPROVED** [review]

**Review by YounixM — APPROVED** [review]

**Pull request merged**